### PR TITLE
Scipy talk

### PR DESCRIPTION
**Pull request by LeJit**
Initial pull request to merge content for the SciPy 2021 tutorial.

Still need to add the Jupyter notebook has been uploaded to the repository.

### CODE REVIEW

File: 2021/scipy/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# SciPy 2021 Tutorial - _Fairness in AI Systems: From Social Context to Practice using Fairlearn_
```
Comment (MiroDudik) [inline 2021/scipy/README.md:1]: Can we have a more shallow path? How about: `2021-scipy-tutorial/`

@fairlearn/fairlearn-maintainers : opinions on this?
I don't expect we will have more than a few talks per year that we want to put in the repo?

File: 2021/scipy/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+For this tutorial, participants will need to download the Jupyter notebook _name goes here_.
+
+In a Python virtual environment (Python version >= 3.7), install the necessary libraries by running
+
+```
+pip install -r requirements.txt
+```
+
+
+### Option 2: Google Colab
```
Comment (MiroDudik) [inline 2021/scipy/README.md:26]: Colab should be the main option for this, local set-up the secondary option.

File: 2021/scipy/scripts/preprocess.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import pandas as pd
```
Comment (MiroDudik) [inline 2021/scipy/scripts/preprocess.py:1]: I think that having subdirectories is an overkill for such a small tutorial, I'd be in favor of keeping everything in one directory. Just like:
https://github.com/propublica/compas-analysis

One possible exception would be `data` folder (with two files in it). For filenames, I'd loosely follow the conventions here:
https://drivendata.github.io/cookiecutter-data-science/#directory-structure

@fairlearn/fairlearn-maintainers : thoughts?

File: 2021/scipy/README.md (hunk 2 above)
Comment (MiroDudik) [reply 2021/scipy/README.md:26]: (but i don't care too strongly)

Comment (romanlutz) [reply 2021/scipy/README.md:26]: Binder?

File: 2021/scipy/README.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1HDiMzQ_P2NHT66OuAq1bBiBfCyNtCXlO)
+
+The other option is to run the notebook through free compute services. Click on the [Google Colab]() badge above to run the notebook through a Google Colab instance.
+
+## Links to Relevant Resources
+
+### Fairlearn
+
+[Fairlearn](www.fairlearn.org) is an open-source toolkit for assessing and mitigating fairness-related harms in machine learning models. Fairlearn provides algorithmns and metrics for assessing and reporting model unfairness and mitigating unfairness when possible.
```
Comment (romanlutz) [inline 2021/scipy/README.md:36]: nit: typo algorithms

Comment (romanlutz) [reply 2021/scipy/README.md:36]: Also, we don't provide algorithms for model assessment AFAIK.

**Review by romanlutz — COMMENTED** [review]
Are we allowed to distribute the data like this?

**Comment by MiroDudik** [comment]
Re. attribution, we can definitely store and adapt the data, we just need to follow CC BY 4.0 standards: https://archive-beta.ics.uci.edu/ml/donation-policy

**Review by MiroDudik — APPROVED** [review]

**Pull request merged**